lms/djangoapps/courseware/management/commands/delete_duplicate_record.py

```diff
@@ -0,0 +1,55 @@
+"""
+Script for deleting duplicate record in courseModuleCompletion table
+for a specific course_id and for a specific content_id and
+for a specific username.
+"""
+
+from django.core.management.base import BaseCommand, CommandError
+from opaque_keys.edx.keys import CourseKey
+from opaque_keys import InvalidKeyError
+from util.prompt import query_yes_no
+from progress.models import CourseModuleCompletion
+from django.contrib.auth.models import User
+
+
+class Command(BaseCommand):
+    """
+    Command class for course reference removal
+    """
+
+    def add_arguments(self, parser):
+        parser.add_argument('args', nargs='*')
+
+    def handle(self, *args, **options):
+        """
+        This handler operation does the actual work!
+        """
+        if len(args) != 3:
+            raise CommandError("delete_record requires exactly three arguments: <course_id> |<content_id>| |username|")
+        try:
+            course_key = CourseKey.from_string(args[0])
+        except InvalidKeyError:
+            raise Exception("Invalid course key")
+
+        content_id = args[1]
+        username = args[2]
+        if username:
+            try:
+                user_id = User.objects.get(username=username).pk
+            except User.DoesNotExist:
+                raise Exception("User does not exists with the given username")
+
+            if content_id:
```

Comment:
one more thing, can we add a console message in case this if clause returns false?